web/templates/stream_settings/stream_members.hbs

```diff
@@ -19,11 +19,11 @@
     <div class="subscriber-list-box">
         <div class="subscriber_list_container" data-simplebar>
             <div class="subscriber_list_loading_indicator"></div>
-            <table class="subscriber-list table table-striped">
+            <table id="stream_members_list" class="subscriber-list table table-striped">
```

Comment:
Why this ID is required? Can't we just use the `subscriber-list` class for `$parent_container`?

Reply:
I initially tried by using `subscriber-list` class as parent but `web/templates/stream_settings/new_stream_users.hbs` also have a table with same class. So when a user has opened stream creation form and then clicks on a stream to view the subscribers there will be two elements with same class and error was shown as I expected only one element.

However on removing `expectOne()` from parent it works fine. So should I remove this and use `subscriber-list` for parent?

Reply:
Ok. Thanks for clarification. We can keep this ID.